### PR TITLE
Al/BE-68. Insurance, deposit and burn.

### PR DESCRIPTION
**Pull request by lut1k**
In the liquidity-pools pallet implemented two extrinsic functions for depositing and burning the balance in the insurance. Wrote unit-tests for the above functionality.

### CODE REVIEW

**Review by Pavel-Blaize — DISMISSED** [review]

File: pallets/liquidity-pools/src/lib.rs [hunk 1]
```diff
@@ -90,36 +95,114 @@ decl_module! {
...
+					Self::deposit_event(Event::RedeemedInsurance(reserve_id, amount));
+					Ok(())
+				})?;
+
+			}
+
+	}
+}
+
+decl_storage! {
```
Comment (Apollo50) [inline pallets/liquidity-pools/src/lib.rs:137]: emm...why did you move Storage under the module?

**Review by Apollo50 — APPROVED** [review]

File: pallets/liquidity-pools/src/lib.rs (hunk 1 above)
Comment (lut1k) [reply pallets/liquidity-pools/src/lib.rs]: > emm...why did you move Storage under the module?

Fixed, bro
